.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   build:
     working_directory: /go/src/github.com/Financial-Times/upp-aggregate-healthcheck
     docker:
-      - image: circleci/golang:1.11
+      - image: circleci/golang:1
```

Comment:
Should be "golang:1" only.

Reply:
https://circleci.com/gh/Financial-Times/upp-aggregate-healthcheck/464

This happens when I use `golang:1`

Reply:
The "Make go get work" step should be removed.